### PR TITLE
add .ms-persona.is-selected

### PR DESCRIPTION
**Pull request by SteenMolberg**
Currently doing the OrgChart directive for [ngOfficeUiFabric ](https://github.com/ngOfficeUIFabric/ng-officeuifabric)and it would be nice to able to select a person or persons in the OrgChart just like in a tablerow.

### CODE REVIEW

**Comment by mikewheaton** [comment]
Thanks for submitting this PR, @SteenMolberg. We just need to check in with our design team and have a quick discussion about whether the Persona should be interactive, or if it's just a display template to be used within other controls.

**Comment by mikewheaton** [comment]
@SteenMolberg: Could you send us a screenshot of the OrgChart with a person selected? We just want to see what this looks like in context.

**Comment by SteenMolberg** [comment]
Below screenshots...

Image1: Items not selectable. Please not the presence icon placement compared to image2
Image2: 2 items selected and mouse hovers over a third item.

officeuifabric1
officeuifabric2

**Comment by ericthompson** [comment]
Thanks, @SteenMolberg! We're passing this along and we'll get back to you as soon as we can.

**Comment by mikewheaton** [comment]
We've discussed this with the design team and they agree that Persona should have a selected state. We just need to review the official designs to be sure this is in line before we can merge it.

**Comment by SteenMolberg** [comment]
thx for info Mike

File: src/components/Persona/Persona.scss [hunk 1]
```diff
@@ -25,11 +25,16 @@ $ms-color-initials-darkRed: 		#b91d47;
 
 
 .ms-Persona {
-	@include ms-font-m;
-	@include ms-u-normalize;
-	display: table;
-	line-height: 1;
-	position: relative;
+  @include ms-font-m;
```
Comment (mikewheaton) [inline src/components/Persona/Persona.scss]: Looks like the indentation has changed on these lines. Could you revert these lines to how they originally were?

File: src/components/Persona/Persona.scss [hunk 2]
```diff
@@ -25,11 +25,16 @@ $ms-color-initials-darkRed: 		#b91d47;
...
+  @include ms-u-normalize;
+  display: table;
+  line-height: 1;
+  position: relative;
+
+  // Persone can be selected.
+  &.is-selected {
+    background-color: $ms-color-themeLight;
```
Comment (mikewheaton) [inline src/components/Persona/Persona.scss]: I found some design documentation that shows we should be using $ms-color-themeLighter. Could you please change it?

**Comment by mikewheaton** [comment]
Apologies for the delays here, @SteenMolberg. I've added a couple comments – could you address these and add a new commit to your pull request?

**Comment by mikewheaton** [comment]
Adding the needs design flag here so that we can verify the selected state.

**Comment by mikewheaton** [comment]
I've opened a [new issue](https://github.com/OfficeDev/Office-UI-Fabric/issues/616) to request official designs for these states. We appreciate this PR but it seems to have been abandoned, and with the recent changes for Fabric 3.0 coming in it no longer merges without conflicts.

**Pull request closed without merging**